eva-accession-release-automation/publish_release_to_ftp/publish_release_files_to_ftp.py

```diff
@@ -111,48 +111,49 @@ def get_folder_path_for_species(release_folder_base, species_release_folder_name
     return os.path.join(release_folder_base, by_species_folder_name, species_release_folder_name)
 
 
-def create_symlink_to_assembly_folder_from_species_folder(current_release_assembly_info, release_properties,
+def get_folder_path_for_species_assembly(release_folder_base, species_release_folder_name, assembly_accession):
+    return os.path.join(release_folder_base, by_species_folder_name, species_release_folder_name, assembly_accession)
+
+
+def create_symlink_to_species_folder_from_assembly_folder(current_release_assembly_info, release_properties,
                                                           public_release_assembly_folder):
-    """
-    Create linkage to assembly folders from the species folder
-    See FTP layout linkage between by_assembly and by_species folders in the link below:
-    https://docs.google.com/presentation/d/1cishRa6P6beIBTP8l1SgJfz71vQcCm5XLmSA8Hmf8rw/edit#slide=id.g63fd5cd489_0_0
-    """
     assembly_accession = current_release_assembly_info["assembly_accession"]
     species_release_folder_name = current_release_assembly_info["release_folder_name"]
-    public_release_species_folder = os.path.join(release_properties.public_ftp_current_release_folder,
-                                                 by_species_folder_name, species_release_folder_name)
-    run_command_with_output("Creating symlink from species folder {0} to assembly folder {1}"
-                            .format(public_release_species_folder, public_release_assembly_folder),
-                            'bash -c "cd {1} && ln -sfT {0} {1}/{2}"'.format(
-                                os.path.relpath(public_release_assembly_folder, public_release_species_folder),
-                                public_release_species_folder, assembly_accession))
+    public_release_assembly_species_folder = os.path.join(public_release_assembly_folder, species_release_folder_name)
+    public_release_species_assembly_folder = os.path.join(release_properties.public_ftp_current_release_folder,
+                                                 by_species_folder_name, species_release_folder_name, assembly_accession)
+    run_command_with_output(f"""Creating symlink from assembly folder {public_release_assembly_species_folder} to 
+                            species folder {public_release_species_assembly_folder}""",
+                            'bash -c "cd {0} && ln -sfT {1} {2}"'.format(public_release_assembly_folder,
+                                os.path.relpath(public_release_species_assembly_folder, public_release_assembly_folder),
+                                public_release_assembly_species_folder))
 
 
 def recreate_public_release_assembly_folder(assembly_accession, public_release_assembly_folder):
```

Comment:
Maybe rename the method & params (and maybe change the logging as well) to make it clear that this is removing the species/assembly folder, not the entire assembly folder

Reply:
updated